webpack/components/OrganizationCheck/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col } from 'patternfly-react';
+import './index.scss';
+
+const OrganizationCheck = (props) => {
+  const { children, hide } = props;
+  if (hide) {
+    return (
+      <Grid bsClass="container-fluid">
+        <Row>
+          <Col sm={12}>
+            <div class="centered-message">
+              <h2>No organization selected</h2>
+              <h3>
+                The page you are attempting to access requires selecting a specific organization.
```

Comment:
The way this reads is that there is only one correct Organization to choose. Can you change the text to something like:
"...requires a selected organization"